### ISSUE TITLE
Support for Riot Games

### ISSUE THREAD

**Issue by Sysadminfromhell** [tpill90/lancache-prefill-common#4]
is there a prefill project planned for riot or even gog planned (I asked for support in Lancache for GOG).

Kind regards,
PS.: I love and admire your other prefill projects, the work just fine!

**Comment by tpill90** [comment]
Nope, I don't have any plans for a riot or gog prefill, or any other prefills for that matter. I do have a working prototype for Xbox gamepass, however I don't have the time to finish it. If there's anyone out there that would help me with finishing it I would certainly appreciate it. 

And I always love to hear the positive feedback! Glad you have found the prefills helpful.

**Comment by Sysadminfromhell** [comment]
> 

I wish I could help but I'm just familiar with PHP/JScript,JavaScript and PowerShell/Batch.
